homeassistant/components/owntracks/__init__.py

```diff
@@ -91,6 +91,24 @@
     return True
 
 
+async def async_unload_entry(hass, entry):
+    """Unload an OwnTracks config entry."""
+    hass.components.webhook.async_unregister(entry.data[CONF_WEBHOOK_ID])
+    await hass.config_entries.async_forward_entry_unload(
```

Comment:
This isn't supported yet.

Reply:
The code will thus return False, which will tell the user that it requires a restart. Once it is supported, it will magically start working